toolchain/lex/tokenized_buffer_test.cpp

```diff
@@ -1107,6 +1107,20 @@ TEST_F(LexerTest, DiagnosticUnrecognizedChar) {
   compile_helper_.GetTokenizedBuffer("\b", &consumer);
 }
 
+TEST_F(LexerTest, DiagnosticFileTooLarge) {
+  Testing::MockDiagnosticConsumer consumer;
+  static constexpr size_t NumLines = 10'000'000;
+  std::string input;
+  input.reserve(NumLines * 3);
+  for (int _ [[maybe_unused]] : llvm::seq(NumLines)) {
```

Comment:
```suggestion
  for ([[maybe_unused]] int _ : llvm::seq(NumLines)) {
```

I assume order doesn't matter to the compiler, but glancing at other locations, I think we're doing this order.